windows/hardening/AD-Hardening.ps1

```diff
@@ -1,6 +1,11 @@
 Import-Module ActiveDirectory
 Import-Module GroupPolicy
 
+if (-not (Test-Path ".\advancedAuditing.ps1") -or -not (Test-Path ".\ports.json") -or -not (Test-Path ".\users.txt")) {
```

Comment:
why not try to download them in the script?

Reply:
Fixed in new push